src/pages/Landing/Landing.jsx

```diff
@@ -0,0 +1,40 @@
+import "./Landing.css";
+import { useState } from "react";
+import { useData } from "../../context/data-context";
+import { Background } from "../../components";
+import { ACTION_TYPE } from "../../utils/constants";
+
+export const Landing = () => {
+  const { dataDispatch } = useData();
+  const [name, setName] = useState("");
+  const nameUpdateHandler = () => {
+    dataDispatch({ type: ACTION_TYPE.ADD_USER, payload: { userName: name } });
+  };
+  return (
+    <>
+      <Background />
+      <div className="main-section p-abs top-btm-lft-rgt fx-col fx-ai-center fx-jc-center">
+        <div className="prompt  fx-col fx-ai-center fx-jc-center">
+          <p className="question-text">Hello, What's your name ?</p>
+          <input
```

Comment:
if you put this input field inside a form, you wont have to explicitly check for enter key event

Reply:
Thanks @shraddha-1402 for suggesting this change, I have updated it in my code.